jsx/PaginationLinks.js

```diff
@@ -1,148 +1,135 @@
-/* exported RPaginationLinks */
+import React, {useState} from 'react';
 
-import React, {Component} from 'react';
-import PropTypes from 'prop-types';
+const theme = {
+  primary: '#053665',
+  secondary: '#A6D3F5',
+  accent: '#E89A0C',
+  neutral: '#DDD',
+};
 
 /**
- * Pagination component
+ * Produces the pagination number links
+ *
+ * @param {jsx} children
+ * @param {bool} active
+ * @param {func} onClick
+ *
+ * @return {jsx}
  */
-class PaginationLinks extends Component {
-  /**
-   * @constructor
-   * @param {object} props - React Component properties
-   */
-  constructor(props) {
-    super(props);
-
-    this.state = {
+function PaginationLink({
+  children,
+  active,
+  onClick,
+}) {
+  const [hover, setHover] = useState(false);
+  const hoverOn = () => setHover(true);
+  const hoverOff = () => setHover(false);
+  const style = {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    margin: '0.2rem',
+    width: '3rem',
+    height: '3rem',
+    background: onClick && hover && !active && theme.neutral,
+    border: active && '0.1rem solid '+theme.primary,
+    borderRadius: '50%',
+    cursor: onClick && 'pointer',
+  };
+
+  return (
+    <>
```

Comment:
@HenriRabalais I think you don't need the `<></>` since there's only one div.

Reply:
Yeah, you're right. I believe I added it because I was checking some styling issues, and was hoping that would fix it. I'll remove it!